src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] supplyAndBuy = readData(fromFileName);
+        int result = processStatistics(supplyAndBuy[0], supplyAndBuy[1]);
+        writeData(toFileName, supplyAndBuy[0], supplyAndBuy[1], result);
+    }
+
+    private int[] readData(String fromFileName) {
```

Comment:
Consider using a custom object to store the 'totalSupply' and 'totalBuy' rather than an array. This would improve the readability of your code by providing more informative names to the data structure.